atlasdb-container-test-utils/src/test/java/com/palantir/atlasdb/containers/ContainersTest.java

```diff
@@ -0,0 +1,39 @@
+/**
+ * Copyright 2016 Palantir Technologies
+ *
+ * Licensed under the BSD-3 License (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://opensource.org/licenses/BSD-3-Clause
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.palantir.atlasdb.containers;
+
+import org.junit.Test;
+
+@SuppressWarnings("checkstyle:IllegalThrows")
+public class ContainersTest {
+    @Test
+    public void containerStartsUpCorrectly() throws Throwable {
+        Containers containers = new Containers(ContainersTest.class)
+                .with(new FirstNginxContainer());
+
+        containers.before();
```

Comment:
I don't like the naming of `before()`, I'm really confused as to what this line is doing. Is it some standard nomenclature? What is this call achieving and why is it here?